src/autosuggest/interfaces.ts

```diff
@@ -160,6 +165,51 @@ export namespace AutosuggestProps {
      */
     select(): void;
   }
+
+  export interface Style {
+    root?: {
```

Comment:
Should we also support styling for the clear input button?

<img width="879" height="151" alt="Image" src="https://github.com/user-attachments/assets/26921506-17e3-4774-9f71-390c42013edc" />

Reply:
Yeah I think so. I also think it should be possible to style the left icon. But the same applies to the regular input + search variant. I'd suggest I write a proposal for that, what do you think?